redcap_bridge/utils.py

```diff
@@ -277,6 +287,8 @@ def parse_choices(choice_str, annotation_str):
             else:
                 warnings.warn(f'Could not determine default choice for {annotation_str}')
 
-    return list(choice_labels), default_choice_label
+    choice_labels = [re.sub(r'{|}', '', label) for label in choice_labels]
```

Comment:
Actually it would be best to not only remove the `{` and `}`, but anything that is wrapped in there. Could you adjust the regex to remove anything that surrounded by `{...}`?